.github/workflows/CI.yaml

```
@@ -68,7 +68,7 @@ jobs:
       - name: Run Tests
         shell: bash -l {0}
         run: |
-          pytest -v --cov=openff --cov-config=setup.cfg openff/bespokefit/tests/ --cov-report=xml
+          pytest -v --cov=openff --cov-config=setup.cfg openff/bespokefit/tests/executor/services/coordinator/test_app.py::test_get_optimization --cov-report=xml
```

```suggestion
          pytest -v --cov=openff --cov-config=setup.cfg openff/bespokefit/tests --cov-report=xml
```

I think this was done to speed up tests? I'm going to remove it to make sure we still get the green check but documenting it in case its important.